packages/auth-client/src/messages/verify.test.ts

```diff
@@ -159,37 +155,14 @@ describe("verify", () => {
       acceptAuthAddress: true,
       getFid,
       isValidAuthAddress,
-      provider,
     });
     expect(result.isOk()).toBe(true);
     expect(result._unsafeUnwrap()).toEqual({
       authMethod: "authAddress",
       data: siweMessage,
-      success: true,
-      fid: 1234,
-    });
-  });
-
-  test("1271 signatures fail without provider", async () => {
```

Comment:
we now will attempt to validate using a default public client connecting to a public rpc url